packages/browser-utils/src/instrument/history.ts

```diff
@@ -19,29 +19,26 @@ export function addHistoryInstrumentationHandler(handler: (data: HandlerDataHist
 }
 
 function instrumentHistory(): void {
-  if (!supportsHistory()) {
-    return;
-  }
-
-  const oldOnPopState = WINDOW.onpopstate;
-  WINDOW.onpopstate = function (this: WindowEventHandlers, ...args: unknown[]) {
+  // This may also be triggered on `pushState`, but it may not always reliably be emitted by the browser
```

Comment:
l: Just to confirm I understand this correctly?
```suggestion
  // The 'popstate' callback may also be triggered on `pushState`, but it may not always reliably be emitted by the browser
```

Reply:
```suggestion
  // The `popstate` event may also be triggered on `pushState`, but it may not always reliably be emitted by the browser
```

actually this :D